src/platform/site-wide/cta-widget/CallToActionAlert.jsx

```diff
@@ -18,10 +20,19 @@ export default function CallToActionAlert({
       <div>
         {alertText}
         {buttonText && (
-          <button className={buttonClass} onClick={buttonHandler}>
+          <button
+            className={buttonClass}
+            onClick={buttonHandler}
+            style={secondaryLinkText ? { marginRight: '1em' } : {}}
```

Comment:
Inline styling is probably not ideal but did not see an appropriate .scss file to add this style to.

Reply:
I think if you make the following `a` a `button`, it should rework the margins appropriately. If not, I think we should be using `rem` units instead of `em`. And the style should be applied on the left margin of the secondary instead of the primary, so there won't be unnecessary margins when only the primary appears.